libindy/src/services/pool/networker.rs

```diff
@@ -343,13 +343,10 @@ impl PoolConnection {
 impl RemoteNode {
     fn connect(&self, ctx: &zmq::Context, key_pair: &zmq::CurveKeyPair) -> IndyResult<ZSocket> {
         let s = ctx.socket(zmq::SocketType::DEALER)?;
-        s.set_identity(key_pair.public_key.as_bytes())?;
+        s.set_identity(&key_pair.public_key)?;
         s.set_curve_secretkey(&key_pair.secret_key)?;
         s.set_curve_publickey(&key_pair.public_key)?;
-        s.set_curve_serverkey(
-            zmq::z85_encode(self.public_key.as_slice())
-                .to_indy(IndyErrorKind::InvalidStructure, "Can't encode server key as z85")? // FIXME: review kind
-                .as_str())?;
+        s.set_curve_serverkey(self.public_key.as_slice())?;
```

Comment:
The z85 just shrinks the number of bytes. So what is the reason this was removed?